block.go

```diff
@@ -162,6 +164,7 @@ func (b *EthBlock) MarshalJSON() ([]byte, error) {
 	return json.Marshal(out)
 }
 
+// Cid returns the content identifier of the block header.
```

Comment:
We usually just abbreviate "content identifier" as CID (kind of like how nobody calls http "hyper-text transport protocol").